File: Backend-POS/POS.Main/POS.Main.Business.Authorization/Models/Permission/PermissionMatrixResponseModel.cs
namespace POS.Main.Business.Authorization.Models.Permission;

public class PermissionMatrixResponseModel
{
    public int PositionId { get; set; }
    public string PositionName { get; set; } = string.Empty;
    public List<int> GrantedAuthorizeMatrixIds { get; set; } = new();
    public ModuleTreeResponseModel ModuleTree { get; set; } = new();
}

File: Frontend-POS/RBMS-POS-Mobile-Web/src/app/core/api/models/permission-matrix-response-model.ts
/* tslint:disable */
/* eslint-disable */
import { ModuleTreeResponseModel } from '../models/module-tree-response-model';
export interface PermissionMatrixResponseModel {
  grantedAuthorizeMatrixIds?: Array<number> | null;
  moduleTree?: ModuleTreeResponseModel;
  positionId?: number;
  positionName?: string | null;
}
